examples/phoenix_signaling/README.md

```diff
@@ -0,0 +1,56 @@
+# PhoenixSignaling
+
+To start your Phoenix server:
+
+  * Run `mix setup` to install and setup dependencies
+  * Start Phoenix endpoint with `mix phx.server` or inside IEx with `iex -S mix phx.server`
+
+Now you can visit [`localhost:4000`](http://localhost:4000) from your browser.
+You should be able to see a video player displaying video captured from your camera.
+
+## How to use PhoenixSignaling in your own Phoenix project?
+
+1. Create new socket in your application endpoint, using the `Membrane.WebRTC.PhoenixSignaling.Socket`, for instance at `/signaling` path:
+```
+  socket "/signaling", Membrane.WebRTC.PhoenixSignaling.Socket,
+  websocket: true,
+  longpoll: false
+```
+2. Create a Phoenix signaling channel with desired signaling ID, for instance in your controller:
+```
+signaling = Membrane.WebRTC.PhoenixSignaling.new("<signaling_id>")
+```
+
+>Please note that `signaling_id` is expected to be unique for each WebRTC connection about to be
+>estabilished. You can, for instance:
+>1. Generate unique id with `:uuid` package and assign to the connection in the page controller:
+>```
+>unique_id = UUID.uuid4()
+>render(conn, :home, layout: false, signaling_id: unique_id)
```

Comment:
Let's mention that we have to start `PhoenixSignaling` and WebRTC Element (or Boombox) as well

Reply:
done